### DISCUSSION TITLE
SportsNet New York

### DISCUSSION THREAD

**Discussion by desmondc474** [iptv-org/community#349] [Show and tell]
### Channel Name

SportsNew New York

### Channel ID

SportsNetNewYork.us

### Stream URL

https://list.iptvcat.com/my_list/s/316b6fe06184e9a210145d12035bc5eb.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Xtream-Codes server. We do not allow these kinds of links.
![image](https://user-images.githubusercontent.com/30985701/169487403-d3e8aa5d-6601-40cf-a0a2-6a8452a1579e.png)

**Comment by takethis2165** [comment]
I had a problem with my VLC,now it's ok.Thanks for the channel

**Comment by takethis2165** [comment]
The link is broken was working before!

**Comment by freearhey** [comment]
https://list.iptvcat.com/my_list/s/316b6fe06184e9a210145d12035bc5eb.m3u8 (offline)
<img width="544" alt="image" src="https://user-images.githubusercontent.com/7253922/232227408-16c28d10-3504-47b4-ad58-a21b6dadd793.png">
